packages/@jsii/integ-test/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@jsii/integ-test",
+  "version": "0.21.2",
+  "description": "A suite of integration tests for JSII and related modules.",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build",
+    "test:integ": "jest"
+  },
+  "keywords": [],
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com"
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@octokit/rest": "^16.36.0",
+    "dotenv": "^8.2.0",
+    "fs-extra": "^8.1.0",
```

Comment:
Odd this doesn't come with `@types/fs-extra`, probably also drue for `tar` & `dotenv`.